doc/ref_model/chapters/chapter07.rst

```diff
@@ -801,6 +801,124 @@ Security certification should encompass the following elements:
   security analysis tools.
 - Resiliency tests run (such as hardware failures or power failure tests)
 
+
+Cloud Infrastructure Regulatory Compliance 
+------------------------------------------
+Evolving cloud adoption in the telecom industry, now encroaching on its inner sanctum of network services, undoubtedly brings many benefits for the network operators 
+and their partners, and ultimately to the end consumers of the telecommunication services. However, it also brings compliance challenges that can seem overwhelming. 
+The telecommunication industry players can reduce this overwhelm by arming themselves with information about which laws they need to comply with, why, and how.  
+
+The costs of non-compliance can be very serious. Organisations may not only have to contend with hefty fines and possible lawsuits, but they may also even end up 
+losing their reputation and eventually losing customers, with an obvious adverse impact on revenues and profitability. 
+
+Compliance means that an operator’s systems, processes, and workflows align with the requirements mandated by the regulatory regimes imposed by the relevant 
+governmental and industry regulatory bodies. The need for compliance extends to the cloud, so operators must ensure that any data transferred to and out, and stored 
+in their cloud infrastructure complies with all relevant data protection and privacy laws.
```

Comment:
Suggest add of "location"
```suggestion
in their cloud infrastructure complies with all relevant data protection, including data residency, and privacy laws.
```

Reply:
@pgoyal01 @walterkozlowski typically when discussing data location (residency) in cloud industry this is covered as "data sovereignty".